Tasks/NotationV0/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "notation",
+  "version": "0.1.0",
+  "description": "Azure Pipepine Task for setting up Notation CLI, sign and verify with Notation",
+  "main": "src/index.js",
+  "scripts": {
+    "build": "tsc -b"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "@types/node": "^10.17.0",
```

Comment:
I believe types/node should be ^16.11.39 for node16

Reply:
Updated to use node 16